### PR TITLE
GitBook: [nosql/mongodb-modules] 1 page modified

### LINKED ISSUE #9: [ROADMAP] Modules for NoSQL/MongoDB
### Pre-Approval
- [ ] Asked a head (Kevin) to give you write access to GitBooks
- [ ] For each module: 
  - [ ] 3-4 relevant learning objectives
  - [ ] 4-5 sample activities with a one-sentence description
  - [ ] 2-3 sample labs with a one-sentence description
  - [ ] (if applicable) modules on road map before it and after it  
- [ ] GitBook Variant/GitHub branch set up with your modifications in the page "Roadmap Topic Breakdown"
- [ ] Content all correctly formatted

### Post-Approval
- [ ] Placing modules appropriately within the Lucidchart road-map

Please consult our handbook [here](https://about.bitproject.org/teams/developer-relations/development-roadmap) for any clarifications or questions!

### PR DESCRIPTION
**Pull request by QuantumOverture**
#### Issues Closed 

- [ ] Closes #9 

...

Changes proposed in this pull request:
* Addition of a RoadMap for NoSQL/MongoDB 

@reviewer/kavuong

### CODE REVIEW

File: teams/developer-relations/development-roadmap/roadmap-topic-breakdown.md [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
-**Module: Intermediate Python**
+**Module: Intro to NoSQL/MongoDB**
 
 _**Learning Objectives**_
 
-* Students will continue applying their fundamental Python skills learned from the previous module.
-* Students will begin learning about how to integrate basic data structures like lists and dictionaries to add complexity to their Python code.
-* Students will learn how they can use for/while loops and if statements to control the flow of their code.
-* Students will learn how to represent strings in Python using F-strings.
+* Students will learn the main difference between NoSQL and SQL databases.
```
Comment (kavuong) [inline teams/developer-relations/development-roadmap/roadmap-topic-breakdown.md]: I think the priority here should be the concepts of NoSQL itself, not focusing on SQL at all.

Comment (kavuong) [inline teams/developer-relations/development-roadmap/roadmap-topic-breakdown.md]: additionally, there should be some learning objectives about MongoDB's various products - Stitch, Atlas, etc. The MongoDB workshop can provide more insight, and @jeffchhen could also help with this

File: teams/developer-relations/development-roadmap/roadmap-topic-breakdown.md [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
 
-* Word Translation Calculator
-  * You are going to write a program that calculates how often a word appears in a given string.
-* Basic Calculator
-  * Create a functional calculator in python that can do simple arithmetic operations on two numbers.
-* Tic-Tac-Toe
-  * Write a Tic-Tac-Toe game, and you will play against an AI which chooses random moves.
+* Simple Pokemon Battle
+  * You are going to write a program that sets up a MongoDB database with a couple of Pokemon\(grouping them by type\) and makes two random Pokemon fight!
+* GutenBooks
```
Comment (kavuong) [inline teams/developer-relations/development-roadmap/roadmap-topic-breakdown.md:34]: we should definitely use the Twitter API here haha

**Review by kavuong — CHANGES_REQUESTED** [review]
Left a few comments with regards to content below! also please restore the python module

**Comment by QuantumOverture** [comment]
I have updated this branch please take a look!

**Pull request merged**